docs/user/dashboard/lens-advanced.asciidoc

```diff
@@ -312,34 +324,33 @@ image::images/lens_percent_chage.png[Bar chart with percent change in sales betw
 
 [discrete]
 [[view-customers-over-time-by-continents]]
-=== Create a table of customers by category over time
+=== Analyze data in a table
 
-Tables are useful when you want to display the actual field values.
-You can build a date histogram table, and group the customer count metric by category, such as the continent registered in user accounts.
+Use tables when you want to display field values.
```

Comment:
do you want to say more here?  tables are good for comparing data.

Reply:
This should't even be here. The reasons and benefits for using tables is in more appropriate topics.